app/db/crud.py

```diff
@@ -19,7 +19,8 @@ def get_users(db: Session,
               limit: int = None,
               username: str = None,
               status: Union[UserStatus, list] = None,
-              admin: Admin = None):
+              admin: Admin = None,
+              reset_strategy: Union[UserDataLimitResetStrategy, list] = None) -> list[User]:
```

Comment:
Please change `list[User]` to `typing.List[User]` to be compatible with Python3.8 too.